tests/Sharpliner.Tests/AzureDevOps/Model/Tasks/TaskBuilderTests.cs

```diff
@@ -155,7 +155,7 @@ private class PwshTaskPipeline : TestPipeline
                         {
                             Pwsh.FromResourceFile("Sharpliner.Tests.AzureDevOps.Resources.Test-Script.ps1"),
                             Pwsh.FromResourceFile("Test-Script.ps1", "A display name"),
-                            Pwsh.Inline("Connect-AzContext", "Set-AzSubscription --id foo-bar-xyz"),
```

Comment:
this was misleading as in order to use the azure-sdk we need to use [AzurePowerShell@5](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/azure-powershell-v5?view=azure-pipelines)